translate/rus.json

```diff
@@ -360,6 +360,7 @@
 "shake":"сотрясение",
 "shortcut 1":"ярлык 1",
 "shortcut 2":"ярлык 2",
+"shortcut 3":"ярлык 3",  
```

Comment:
I somehow found out you can add a 3rd shortcut and added this translation since you forgot it. Enjoy!